dnf/dnf-main.c

```diff
@@ -58,9 +59,38 @@ process_global_option (const gchar  *option_name,
     }
   else if (g_strcmp0 (option_name, "--setopt") == 0)
     {
-      if (g_strcmp0 (value, "tsflags=nodocs") == 0)
+      g_auto(GStrv) setopt = g_strsplit (value, "=", 2);
+      if (!setopt[0] || !setopt[1])
         {
-          opt_nodocs = TRUE;
+          local_error = g_error_new (G_OPTION_ERROR,
+                                     G_OPTION_ERROR_BAD_VALUE,
+                                     "Missing value in: %s", value);
+          ret = FALSE;
+        }
+      else if (g_strcmp0 (setopt[0], "tsflags") == 0)
```

Comment:
Why you use g_strcmp0? The test for null was already done. (minor)
It is valid for multiple usage of g_strcmp0.

Reply:
OK. I replaced it using strcmp.